rules/integrations/azure/persistence_mfa_disabled_for_azure_user.toml

```
@@ -2,36 +2,39 @@
 creation_date = "2020/08/20"
 integration = ["azure"]
 maturity = "production"
-updated_date = "2024/05/21"
+updated_date = "2025/08/20"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies when multi-factor authentication (MFA) is disabled for an Azure user account. An adversary may disable MFA
+Identifies when multi-factor authentication (MFA) is disabled for an Entra ID user account. An adversary may disable MFA
 for a user account in order to weaken the authentication requirements for the account.
 """
-from = "now-25m"
-index = ["filebeat-*", "logs-azure*"]
+from = "now-9m"
+index = ["filebeat-*", "logs-azure.auditlogs-*"]
 language = "kuery"
 license = "Elastic License v2"
-name = "Multi-Factor Authentication Disabled for an Azure User"
+name = "Multi-Factor Authentication Disabled for User"
```

For consistency across the rulesets could you add the datasource to the beginning of the name? Across the Azure ruleset I've seen a mix of "Microsoft Entra ID" and "Azure Entra" used, not sure which is appropriate here. 
```suggestion
name = "Microsoft Entra ID Multi-Factor Authentication Disabled for User"
```